composer.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "joomla/coding-standard",
+    "description": "Joomla! Coding Standard",
+    "keywords": ["joomla", "coding standard", "phpcs"],
+    "homepage": "https://github.com/joomla/coding-standards",
+    "license": "GPL-2.0+",
+    "require": {
+        "php": ">=5.3.10",
+        "squizlabs/php_codesniffer": "1.*"
```

Comment:
AFAIK `1.*` is similar to `~1.0` (unless I misread something somewhere).  Either way we can definitely update this to whatever folks agree with.